CONTRIBUTORS.txt

```diff
@@ -0,0 +1,161 @@
+# This file is autogenerated by 'contributors-txt',
+# please do not modify manually
+
+Contributors
+------------
+- Claudiu Popa <pcmanticore@gmail.com>
+- LOGILAB S.A. (Paris, FRANCE) <contact@logilab.fr>
+- Sylvain Thénault <thenault@gmail.com>
+- Pierre Sassoulas <pierre.sassoulas@gmail.com>
+- hippo91 <guillaume.peillex@gmail.com>
+- Marc Mueller <30130371+cdce8p@users.noreply.github.com>
+- Daniël van Noord <13665637+DanielNoord@users.noreply.github.com>
```

Comment:
I was wondering about the sorting. Does it make sense to sort by number of commits? That might cause a more extensive diff.

Maybe just alphabetical would be better? Or two different sections `Team` and `Contributors`? _Section names can be improved probably._

Reply:
I don't mind the little diff that will happen when Daniel surpass us all 😄 

> two different sections Team and Contributors? 

This is what is done for pylint more or less (just really out of date), but we'd need to have some configuration with the list of admin though. Is it worth it ?

Reply:
If you're ok with the diff, then it's fine by me 🤷🏻‍♂️

Reply:
Just FYI I might try to do something a little fancier for pylint to preserves the comments about what was done by whom but I did not do anything yet. I'm trying to release astroid first 😄